decidim-core/app/queries/decidim/organization_participatory_processes.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+module Decidim
+  # This query class returns the visible Participatory Processes given an
+  # Organization.
+  class OrganizationParticipatoryProcesses < Rectify::Query
```

Comment:
It's a Query, not a command! It there's Queries inside `commands`, then we should most definitely move that!